### DISCUSSION TITLE
Welcome to hacker-laws Discussions!

### DISCUSSION THREAD

**Discussion by dwmkerr** [dwmkerr/hacker-laws#329] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JeromeKharley** [comment]


**Reply by dwmkerr** [reply]
@JeromeKharley please don't advertise random links on this repo - this comment has been reported and will now be deleted.

**Comment by kozzonm** [comment]
Thanks for another informative website. Where else could I get that kind of info written in such an ideal way?
<a href="https://startentrepreneureonline.com/">Start entrepreneur online</a>
<a href="https://www.youtube.com/redirect?q=https://startentrepreneureonline.com/cheapest-web-hosting/">cheapest web hosting</a>
<a href="https://www.youtube.com/redirect?q=https://startentrepreneureonline.com/cheapest-web-hosting/">cheapest web hosting</a>
<a href="https://www.google.com/url?q=https://startentrepreneureonline.com/cheapest-web-hosting/">cheapest web hosting</a>
<a href="https://www.google.com/url?q=https://startentrepreneureonline.com/cheapest-web-hosting/">cheapest web hosting</a>
<a href="https://plus.google.com/url?q=https://startentrepreneureonline.com/cheapest-web-hosting/">cheapest web hosting</a>
<a href="https://google.com/url?q=https://startentrepreneureonline.com/">entrepreneur</a>
<a href="https://google.ac/url?q=https://startentrepreneureonline.com/">entrepreneur</a>
<a href="https://google.ad/url?q=https://startentrepreneureonline.com/">entrepreneur</a>
<a href="https://google.ae/url?q=https://startentrepreneureonline.com/">entrepreneur</a>
<a href="https://google.com.af/url?q=https://startentrepreneureonline.com/">entrepreneur</a>
<a href="https://google.com.ag/url?q=https://startentrepreneureonline.com/">entrepreneur</a>
<a href="https://google.com.ai/url?q=https://startentrepreneureonline.com/">entrepreneur</a>
<a href="https://google.al/url?q=https://startentrepreneureonline.com/">entrepreneur</a>

**Reply by dwmkerr** [reply]
Reporting this content as it is spam. Will delete now, please don't spam on this project.

**Comment by Joshua-rose** [comment]
Just curious. Is there any tool that gives you a random law each call?

**Reply by umutphp** [reply]
Hi @Joshua-rose ,

I have a CLI to do it :). Please check it. 
[hacker-laws-cli](https://github.com/umutphp/hacker-laws-cli)

**Reply by Sam241onyia** [reply]
Hey I'm new here
